plugins/git/git.plugin.zsh

```diff
@@ -32,6 +32,22 @@ function work_in_progress() {
     echo "WIP!!"
   fi
 }
+#change current branch old email with new eamil
+gmvemail() {
+  export GMVEMAIL_GIT_EMAIL_OLD=$1
+  export GMVEMAIL_GIT_EMAIL_NEW=$2
```

Comment:
Local var doesn't work here, I changed name to avoid modifying env var by accident